lib/simplifyExpression/collectAndCombineSearch/ConstantOrPowerTerm.js

```diff
@@ -0,0 +1,45 @@
+const NodeType = require('../../node/Type');
```

Comment:
consider calling this file `ConstantOrConstantPower` even though it's longer? especially since your checker function is `isConstantOrConstantPower`

just because now it seems like it's either a constant term, or a "power" term, where power is kinda ambiguous